package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "brave-crypto",
-  "version": "0.2.3",
+  "version": "0.2.4",
```

Comment:
do you need me to do a release to the npm registry? if not i'd prefer to keep the version number the same but no big deal either way

Reply:
no update needed since `brave-core` is referencing the hashes via `DEPS` file 👍

Reply:
@bsclifton could you please remove this line then? (keep the version at 0.2.3)

Reply:
@diracdeltas done! 😄